packages/@aws-cdk/aws-certificatemanager/README.md

```diff
@@ -45,19 +45,26 @@ records for your domain.
 See [Validate with DNS](https://docs.aws.amazon.com/acm/latest/userguide/gs-acm-validate-dns.html)
 in the AWS Certificate Manager User Guide.
 
-### Automatic DNS-validated certificates using Route53
+```ts
+new Certificate(this, 'Certificate', {
+  domainName: 'hello.example.com',
+  validation: CertificateValidation.fromDns(),
```

Comment:
I feel like the README here should call out the same warning from the `fromDns` documentation about needing to manually add the DNS records before CloudFormation can succeed. Might even be worth having the Route53 example first (as the thing people are most likely to copy/paste and use), and then call out the outside-DNS example with the warning about the need for manual intervention.